### DISCUSSION TITLE
Default rules aliases

### DISCUSSION THREAD

**Discussion by coder2000** [palkan/action_policy#219] [Q&A]
I couldn't find anything in the documentation but like Pundit are there default aliases like if I define `update?` in my policy will it also apply to checks against `edit?`?

**Comment by palkan** [comment]
I think, you're looking for: https://actionpolicy.evilmartians.io/#/aliases

**Reply by coder2000** [reply]
I was reading that but couldn't see anywhere that it was something included. It does say "By default, ActionPolicy::Base adds one alias: alias_rule :new?, to: :create?." Is that the only alias available by default? If I just wanted to define `update?` in my policies I would have to make an alias myself?

**Reply by palkan** [reply]
Here are the defaults included into `ActionPolicy::Base`: https://github.com/palkan/action_policy/blob/master/lib/action_policy/policy/defaults.rb

The `default_rule :manage?` configuration says that all rules not specified explicitly should be handled by the `manage?` rule.
So, if you defined `def manage?`, it would be used for `update?`, `edit?`, `destroy?`, etc. (unless you define them explicitly).
